xlators/storage/posix/src/posix-common.c

```diff
@@ -1254,8 +1245,6 @@ posix_init(xlator_t *this)
 
             GF_FREE(_private->base_path);
 
-            GF_FREE(_private->hostname);
```

Comment:
You need to remove GF_FREE(hostname) from posix_fini also.

Reply:
Argh. Sure.